roles/ngi_pipeline/tasks/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
 
+- name: Add py3 anaconda env path to sourceme script for {{ site }}
```

Comment:
personally, I would put this task after the environment has been created but it's not that important